musicrecs/static/style.css

```diff
@@ -0,0 +1,197 @@
+/*Reset Stylesheet to clear the default formatting across all browsers.*/
+
+
+/* 
+html5doctor.com Reset Stylesheet
+v1.6.1
+Last Updated: 2010-09-17
+Author: Richard Clark - http://richclarkdesign.com 
+Twitter: @rich_clark
```

Comment:
Do we have to credit rich clark here? Haha

Reply:
I'm just trying to stay in good standing with the Open Source Administration.

Reply:
fair enough